packages/styles/scss/components/pricing-table/_pricing-table.scss

```diff
@@ -155,6 +155,7 @@
 
     .#{$prefix}--pricing-table-cell-content {
       display: flex;
+      flex-direction: column;
       align-items: start;
```

Comment:
Are these styles doing anything for us anymore? I disabled them in the browser and didn't see any adverse effects, so I think they're safe to remove.

Reply:
Hey @jkaeser, thanks a lot for reviewing this PR 😁
That's the style that's making all flex items to be horizontally organized. Yeah, removing it in the browser's devtool makes no immediate effect but when I remove it from the source code and rerun the storybook them items get squished again:

![Screenshot 2023-10-11 at 14 49 56](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/assets/7583437/aa144147-a467-4309-be4b-b72d3e56e175)

Reply:
I mean the entire block. Removing all of these resulted in no visual changes for me:
```css
display: flex;
flex-direction: column;
align-items: start;
```
Note the styles are duplicated because of the way the CSS is bundled, so you need to disable them in two places in dev tools.

Reply:
@jkaeser hey john, yes! You were right, I missed that I needed to disable the properties in another place. I've just submitted another commit removing those unnecessary properties.